history-of-quakers/modernized/15-chapter-13.adoc

```diff
@@ -1651,6 +1835,8 @@ So help me God.
 
 WILD.
```

Comment:
Not sure what this means, it kind of looks like a signature. Jason had me take out all the Latin that was in this, so maybe it has something to do with that, idk. Here's the link to the PDF original and it's on page 474 if you want to see what it looks like there. file:///C:/Users/Owner/Documents/MSF%20Work/William%20Sewel/Sewel's%20History.pdf

Reply:
I have no idea either.  Maybe let's mark it as a signature, just because that's what it looks like in the original.